system/logger/wrapper/src/schema.graphql

```diff
@@ -0,0 +1,19 @@
+#import { Module } into Logger from "wrap://ens/js-logger.polywrap.eth"
```

Comment:
This should use the core logger interface and we need to call all the implementations of the interface inside the wrapper.